### DISCUSSION TITLE
How to update

### DISCUSSION THREAD

**Discussion by yiqiangjizhang** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#274] [Q&A]
I wanted to ask how to update the app as you update the code and fix it. Do I need to install the latest version each time?

**Comment by im-coder-lg** [comment]
You must build it yourself. The guides are present, but Electron hasn't been allowing me to build, lately. Can you try the build methods and tell here? I don't know if it's the problem with Electron or my computer.

**Reply by yiqiangjizhang** [reply]
So it will update by itself then?

**Reply by im-coder-lg** [reply]
No, if you get a release, you don't get the latest commit updates, particularly because it's a release. But, if you can build it(it didn't work for me for a long time now), then you can run the latest version, and you have to build again to update to the latest commit. After you build it, you get an installer and 2 versions of preinstalled, coz the last successful build I had had 2 preinstalled and an installer. You can use either, but I recommend the preinstalled version.

**Comment by lucasdeeiroz** [comment]
I don't know why but I can't build it on Windows 10. Can anyone who is able to do it zip it and upload it somewhere?

**Reply by Melvin-Abraham** [reply]
I would recommend going through [Build Instructions (Windows)](https://www.electronjs.org/docs/development/build-instructions-windows), an official Electron Guide for building on Windows.

**Reply by yiqiangjizhang** [reply]
So it will update by itself then?

**Reply by lucasdeeiroz** [reply]
> I would recommend going through [Build Instructions (Windows)](https://www.electronjs.org/docs/development/build-instructions-windows), an official Electron Guide for building on Windows.

no dice

**Reply by Melvin-Abraham** [reply]
@lucasdeeiroz What is the version of Node.js that you have installed on the machine?

**Reply by lucasdeeiroz** [reply]
@Melvin-Abraham it was the latest from their website. I followed all the instructions, but I had some errors and was too much of a pain for a thing I have in my phone. RC2 was eating too much memory and mic was going crazy. I see that those major issues were fixed and deserved an update, and although it's cool to build your own (it's actually fun), it's not worth it to install so many softwares on a not so much powerful machine and debug errors to build GA.

**Reply by Melvin-Abraham** [reply]
@lucasdeeiroz Make sure you have Node.js 12 installed.

> I see that those major issues were fixed and deserved an update

An update is right in the corner. You would soon be able to update the app. The only reason for this delay is because the next release is planned to be a stable one and I really want to make sure that it does not ship with too many bugs.
